README.md

```diff
@@ -133,7 +133,7 @@ Fired after the element is created and attached to the DOM.
 
 #### `onupdate`
 
-Fired after the element attributes are updated. This event will fire even if the attributes have not changed.
+Fired after the existing element is updated. This event does *not* fire with `oncreate`, but *will* fire during element updates, regardless of whether any attributes have actually changed.
```

Comment:
> Fired after the existing element is updated. This event will fire even if the attributes have not changed.

Reply:
I don't understand? You're just quoting the original text?

The point was to clarify the fact that `oncreate` and `onupdate` are mutually exclusive.